packages/gatsby-plugin-feed/README.md

```diff
@@ -8,29 +8,13 @@ Create an RSS feed (or multiple feeds) for your Gatsby site.
 
 ## How to Use
 
-```javascript
```

Comment:
Deleted this, rather than documenting the simple case (which will rarely be used).